chain/x/oracle/keeper/result.go

```diff
@@ -50,3 +50,30 @@ func (k Keeper) AddResult(
 	store.Set(types.ResultStoreKey(id), resultHash)
 	return resultHash, nil
 }
+
+// SetResult set result to the store.
+func (k Keeper) SetResult(ctx sdk.Context, regID types.RequestID, result []byte) {
```

Comment:
Move it to above `AddResult`. Also please name variable `reqID`

Reply:
fixed